VTEX - Price Simulations.json

```diff
@@ -798,16 +905,4 @@
                 }
             }
         }
-    },
-    "tags": [
-        {
-            "name": "Session Management"
-        },
-        {
-            "name": "Custom Prices"
-        },
-        {
-            "name": "Price Association"
-        }
-    ]
-}
```

Comment:
Por que deletar as global tags (linhas 801 a 813)? É uma boa prática mantê-las declaradas aqui, tem até alguns alertas do Spectral sobre isso falando: "Operation tags must be defined in global tags."